extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -239,19 +244,19 @@ public function loadData() {
 					}
 
 					if ( $oUTitle instanceof Title ) {
-						$links[0] = $sk->makeLinkObj( $oUTitle, $wgLang->ucfirst(wfMsg($msg) ) );
+						$links[0] = $sk->makeLinkObj( $oUTitle, $lang->ucfirst(wfMsg($msg) ) );
 					}
 
-					if ( $wgUser->isAllowed( 'block' ) && ( !$userIsBlocked ) ) {
+					if ( $user->isAllowed( 'block' ) && ( !$userIsBlocked ) ) {
 						$links[] = $sk->makeLinkObj(
 							Title::newFromText( "BlockIP/{$oUser->getName()}", NS_SPECIAL ),
-							$wgLang->ucfirst( wfMsg('blocklink') )
+							$lang->ucfirst( wfMsg('blocklink') )
 						);
 					}
-					if ( $wgUser->isAllowed( 'userrights' ) && ( !$userIsBlocked ) ) {
+					if ( $user->isAllowed( 'userrights' ) && ( !$userIsBlocked ) ) {
 						$links[] = $sk->makeLinkObj(
 							Title::newFromText( 'UserRights', NS_SPECIAL ),
-							$wgLang->ucfirst( wfMsg('listgrouprights-rights') ),
+							$lang->ucfirst( wfMsg('listgrouprights-rights') ),
 							"user={$oEncUserName}"
 						);
 					};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)

[project-id]: wikia:app